tests/checker/good/gold/crowdfunding_proc.scilla.gold

```diff
@@ -3,7 +3,7 @@
     "State variables": [
       { "field": "owner", "tag": "NotMoney" },
       { "field": "max_block", "tag": "NotMoney" },
-      { "field": "goal", "tag": "Money" },
```

Comment:
Why did this happen? `goal` should be tagged `Money`, right?

Reply:
Yes, but with this change the comparison with the `goal` field are moved into library functions, which are not checked by the cashflow analyser. So there is no way for it to infer that `goal` represents an amount of tokens until we fix issue #400.